segunolalive/helloBooks
# #153539763 Prevent users from mistakingly deleting books

#### What does this PR do?

adds a modal for confirming delete action

#### Description of Task to be completed?

Admin user should be able to delete book

#### How should this be manually tested?

Log in to application as admin, select a book and click on the delete button
A confirmation modal should be displayed

#### Any background context you want to provide?

Having a confirmation modal ensures that a non-reversible action delete is not performed by mistake

#### What are the relevant pivotal tracker stories?

#153539733
#153539763

#### Screenshots (if appropriate)

![localhost_8080_admin 1](https://user-images.githubusercontent.com/16195512/33804562-0622d722-dda8-11e7-8546-ddacd9d11427.png)
